docs/_docs/model.md

```diff
@@ -442,11 +442,48 @@ This object has the following methods that you can call.
 - `Model.transaction.create`
 - `Model.transaction.delete`
 - `Model.transaction.update`
+- `Model.transaction.conditionCheck`
 
 You can pass in the same parameters into each method that you do for the normal (non-transaction) methods.
 
 These methods are only meant to only be called to instantiate the `dynamoose.transaction` array.
 
+
+### Model.transaction.conditionCheck(key, options)
+
+This method allows you to run a conditionCheck when running a DynamoDB transaction.
+
+Example:
+
+```js
+Model.transaction.conditionCheck("credit1", {
+  condition: "amount > :request",
+  conditionNames: {
+    request: "request"
+  },
+  conditionValues: {
+    request: 100
+  }
+})
+```
+
+#### Options
+
+**condition**: string
+
+An expression for a conditional update. See [the AWS documentation](http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.SpecifyingConditions.html) for more information about condition expressions.
+
+**conditionNames**: object
+
+A map of name substitutions for the condition expression.
+
+**conditionValues**: object
```

Comment:
I wonder... Could we provide an abstraction for this? Where instead of the native AWS interface, we'd use an object for condition names & values? i.e.
```js
Model.transaction.conditionCheck(
  "credit1",
  "amount > :request",
  {
    request: 100
  }
);
```

This would simplify it a bit, and make it easier to work with in the future. WDYT? I realize that this would be inconsistent with how most of the code is written today, though.

Reply:
@jutaz I think that looks good. Although I think if anything, we should do a refactor of the entire condition system for everything (queries, scan, etc). I tried to keep it consistent with those other things for now.

Reply:
Yeah, the query interface could be simplified, I think to allow people avoid learning DynamoDB specifics, yet still be able to use Dynamoose without issues

Reply:
@jutaz Totally agree! I will create a new issue for that. Just solved the merge conflicts. Do you think this is ready to merge in without this suggestion, and maybe tackle that later?

Reply:
Yep, [as noted in the review comment](
https://github.com/dynamoosejs/dynamoose/pull/485#pullrequestreview-180971698). This is something that would need to be tackled globally for this package 👍

Reply:
https://github.com/dynamoosejs/dynamoose/issues/491